### ISSUE TITLE
[Due for payment 2026-04-03] [$250] SR: Settings Security: The status messages is not announced.

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#76928]
## Action Performed:
Prerequisites:
- The user is signed in

1. Using NVDA+Firefox, open the URL – https://new.expensify.com/

2. Navigate using the Tab key to the Account button and press Enter to activate it.
3. Navigate using the Tab key to the Security button and press Enter to activate it.
4. Navigate to the Two-factor authentication section and press Enter to activate it.
5. Enter the verification code that is sent to your registered Email.
6. In the Two-factor authentication modal, navigate to the Copy button/Download button and activate it.
7. Observe the screen reader announcement.

## Expected Result:
When the action is completed, the status message should be programmatically conveyed to assistive technologies using an ARIA live region, allowing screen readers to announce the success message (e.g., “Codes copied to clipboard.”).

## Actual Result:
When the user copies or downloads the code, a visual confirmation message, “Copied!”, appears on the screen, but it is not announced by the screen reader.

HTML:
<div dir="auto" class="css-146c3p1" style="color: rgb(0, 46, 34); font-size: 13px; text-align: center; font-family: &quot;Expensify Neue&quot;, &quot;Windows Segoe UI Emoji&quot;, &quot;Noto Color Emoji&quot;, &quot;Custom Emoji Font&quot;; font-style: normal; font-weight: 700;">Copy&nbsp;</div>

## Area issue was found in:
Settings-Security

## Failed WCAG checkpoints
WCAG 4.1.3 Status Messages
https://www.w3.org/WAI/WCAG22/Understanding/status-messages

## User impact:
When appropriate roles or properties are assigned to status messages, the new content is spoken by screen readers in such a way as to assist blind and low vision users. Assigning proper roles or properties to status messages provides possible future uses and personalization opportunities, such as the potential to be exploited by assistive technologies created for users with some cognitive disabilities.

## Suggested resolution:
Add an ARIA live region to ensure the message is announced dynamically when displayed:

Refer:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Guides/Live_regions

## Environment(s)
Windows Windows 11 - Firefox

## Screenshots/Videos
<img width="1919" height="966" alt="Image" src="https://github.com/user-attachments/assets/b6ec5097-c515-451e-bdbb-144570f8bf5b" />

https://github.com/user-attachments/assets/0a6629d6-7b95-4432-aca5-69562591963a

**Version Number:** NA
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36605/community-issues/6999195
**Issue reported by:** NA
**Slack conversation:** NA

<details><summary>Issue Owner</summary>Current Issue Owner: @mallenexpensify</details>
<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~022031835473570642601</li>
        <li>Upwork Job ID: 2031835473570642601</li>
        <li>Last Price Increase: 2026-03-11</li>
<li>Automatic offers: </li>
<ul>
<li>aimane-chnaif | Contributor | 110876556</li>
</ul></li>
    </ul>
</details>

**Comment by Krishna2323** [comment]
@mallenexpensify this one is ready for payments.

**Comment by mallenexpensify** [comment]
Thx @MonilBhavsar payment summary above is accurate as I initially paid @aimane-chnaif $250

**Comment by Krishna2323** [comment]
Requested in ND.

**Comment by laurenreidexpensify** [comment]
@Krishna2323 $250 approved in New Expensify

**Comment by marufsharifi** [comment]
@mallenexpensify, I have a question about this issue.

I was initially assigned/hired, had already opened a [PR](https://github.com/Expensify/App/pull/82935), tested, and requested review of the PR before it was reassigned due to the deadline and estimation concerns here:

https://expensify.slack.com/archives/C01GTK53T8Q/p1772739593580499

Am I still eligible for compensation in this case? This is a similar case to [here](https://github.com/Expensify/App/issues/77490#issuecomment-4237764927).

Thanks in advance for your guidance!

**Comment by mallenexpensify** [comment]
👀 above @rushatgabhane , thx

**Comment by rushatgabhane** [comment]
Similar here, the solution by @Krishna2323 is different from @marufsharifi's PR. It doesn't use hacks, and uses react native primitives

I know you put in the work, but we didn't merge your PR. So I don't think it is eligible for comp 😅
